[FEATURE] Profile settings page

Wilfred007 opened the pull request:
closes #7

Wilfred007 committed:
settings

jsandinoDev commented on CryptoHackAI-web/package-lock.json:
Remove this file from this PR

jsandinoDev commented on CryptoHackAI-web/src/app/components/settings.tsx:
```diff
@@ -0,0 +1,88 @@
+      <div className="min-h-screen w-full p-4 flex items-start justify-center">
+      <div className="fixed top-[194px] left-[353px] w-[641px] h-[790px] rounded-[60px] bg-white shadow-[0_0_40px_rgba(0,0,0,0.08)]">
+        <div className="p-12 h-full flex flex-col">
+          {/* Header */}
```
Remove Comments

jsandinoDev commented on CryptoHackAI-web/src/app/components/settings.tsx:
```diff
@@ -0,0 +1,88 @@
+            <h1 className="text-3xl font-bold text-gray-900">Profile Settings</h1>
+          </div>
+
+          {/* Form */}
```
Remove Comments

jsandinoDev commented on CryptoHackAI-web/src/app/settings/page.tsx:
```diff
@@ -0,0 +1,90 @@
+    <div className="bg-[#B4AB9A] min-h-screen flex items-center justify-center p-4">
+      <div className="w-full max-w-lg sm:w-3/6 rounded-[40px] sm:rounded-[60px] bg-[#f8fafc] shadow-[0_0_40px_rgba(0,0,0,0.08)] mt-10 mb-10">
+        <div className="p-6 sm:p-12 h-full flex flex-col">
+          {/* Header */}
```
Remove Comments

jsandinoDev commented on CryptoHackAI-web/yarn.lock:
Remove this file from this PR

jsandinoDev commented:
Add a more specific title to the PR

Wilfred007 commented:
Okay

On Fri, Feb 21, 2025 at 6:54 AM Josue Sandino ***@***.***>
wrote:

> Add a more specific title to the PR
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Chonete-Builders/CryptoHack-AI/pull/20#issuecomment-2673538949>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3NIGNFMAFFWC2OZDYK66UT2Q25S7AVCNFSM6AAAAABXSHCY2CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNZTGUZTQOJUHE>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
> [image: jsandinoDev]*jsandinoDev* left a comment
> (Chonete-Builders/CryptoHack-AI#20)
> <https://github.com/Chonete-Builders/CryptoHack-AI/pull/20#issuecomment-2673538949>
>
> Add a more specific title to the PR
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Chonete-Builders/CryptoHack-AI/pull/20#issuecomment-2673538949>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3NIGNFMAFFWC2OZDYK66UT2Q25S7AVCNFSM6AAAAABXSHCY2CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNZTGUZTQOJUHE>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>

Wilfred007 committed:
fix

Wilfred007 committed:
settings page width

jsandinoDev commented on CryptoHackAI-web/yarn.lock:
Remove this file && Change PR description

Wilfred007 commented:
![profile setting](https://github.com/user-attachments/assets/4e7ba46f-0383-4338-955b-329f8206adb9)
I worked on the profile settings page at this comment I've removed all the comments, deleted the packagelock.json file, yarnlock.json as well.

jsandinoDev reviewed (approved):
LGTM!
